orbs/shared/jobs/test.yaml

```diff
@@ -7,6 +7,9 @@ parameters:
     default: _json_key
     description: Username to use when fetching images from a registry
     type: string
+  pre_test_script:
```

Comment:
Given this is ran by the `setup.sh` script, it might make more sense to call this `pre_setup_script`, since this can be run in other contexts than just the `test.sh` script.

Reply:
fixed